src/apple/processor.rs

```diff
@@ -137,7 +137,6 @@ pub fn init_processors(port: ffi::mach_port_t) -> (Processor, Vec<Processor>) {
     let mut processors = Vec::new();
     let mut pourcent = 0f32;
     let mut mib = [0, 0];
-
```

Comment:
Why this change?

Reply:
Was not by intention will revert,  
Probably forgot to revert this change while trying something out.